cfgov/paying_for_college/jinja2/paying-for-college/college-cost-blocks/03-grants-and-scholarships.html

```diff
@@ -0,0 +1,168 @@
+                input.render({
+                    'label': 'Federal SEOG grant',
+                    'id': 'grants__seog',
+                    'value': '0',
+                    'data_attribute': 'data-financial-item="grant_seog"'
+                })
+            }}
+            {{
+                input.render({
+                    'label': 'Other Federal Grants',
+                    'id': 'grants__otherFederal',
+                    'value': '0',
+                    'data_attribute': 'data-financial-item="grant_otherFederal"'
+                })
+            }}
+            {{
+                input.render({
+                    'label': 'State grant(s)',
+                    'id': 'grants__state',
+                    'value': '0',
+                    'data_attribute': 'data-financial-item="grant_state"'
+                })
+            }}
+            {{
+                input.render({
+                    'label': 'School grant(s)',
+                    'id': 'grants__school',
+                    'value': '0',
+                    'data_attribute': 'data-financial-item="grant_school"'
+                })
+            }}
+            <div class="block block__sub">
+                <h4 class="header__border-bottom">Servicemember benefits</h4>
+                {{
+                    input.render({
+                        'label': 'Military Tuition Assistance',
+                        'id': 'grants__mta',
+                        'value': '0',
+                        'data_attribute': 'data-financial-item="grant_mta"'
+                    })
+                }}
+                {{
+                    input.render({
+                        'label': 'GI Bill',
+                        'helper_text': 'We recommend checking out the <a href="https://www.va.gov/gi-bill-comparison-tool/" target="_blank" class="external-link">GI Bill&reg; Comparison Tool</a> too.' | safe,
+                        'id': 'grants__gibill',
+                        'value': '0',
+                        'data_attribute': 'data-financial-item="grant_gibill"'
+                    })
+                }}
+                {{
+                    input.render({
+                        'label': 'Other (Yellow Ribbon, etc)',
+                        'id': 'grants_serviceOther',
+                        'value': '0',
+                        'data_attribute': 'data-financial-item="grantserviceOther"'
+                    })
+                }}
+            </div>
+            {{
+                text_item.render({
+                    'label': 'Total grants',
+                    'value': '$99,999',
+                    'type': 'subtotal',
+                    'data_attribute': 'data-financial-item="total_grants"'
+                })
+            }}
+        </div>
+    </div>
+    <div class="grants-and-scholarships_content-section financial-items">
+        <div class="block block__mid">
+            <h3 class="header__border-bottom">Scholarships</h3>
```

Comment:
Should this include the `header` block?

Reply:
Updated to `u-border-bottom` class in https://github.com/cfpb/cfgov-refresh/pull/5742/commits/9237cac212c52dfd381dd65958ae6f2af5f8570c.